### PR TITLE
Db crud

### PR DESCRIPTION
**Pull request by otaiwo70**
first draft for ongoing work on player using repository pattern

### CODE REVIEW

File: backend/game/player/test_player.py [hunk 1]
```diff
@@ -13,6 +13,21 @@ def player():
...
+    assert player.monster_name == 'Godzilla'
+
+
+def test_set_username(player):
+    player.set_username('Tester')
+    assert player.username == 'Tester'
+
+
+def test_set_password(player):
+    player.set_password('12345678')
```
Comment (cvennel) [inline backend/game/player/test_player.py]: I don't think we should be storing plain text passwords for users. Are we planning to implement authentication? If so l, I think we should handle that outside of the player class. I don't think passwords should go in these database objects and can just be removed for the scope of this PR. If we used some external authentication provider (like sign in with google) we wouldn't even need passwords I would think

Comment (otaiwo70) [reply backend/game/player/test_player.py]: password reference removed

File: backend/game/migrations/0001_initial.py [hunk 2]
```diff
@@ -59,7 +61,8 @@ class Migration(migrations.Migration):
                 ('monster_name', models.CharField(max_length=30)),
                 ('username', models.CharField(max_length=30)),
                 ('password', models.CharField(max_length=30)),
```
Comment (kaubster) [inline backend/game/migrations/0001_initial.py]: remove passwords?

Comment (otaiwo70) [reply backend/game/migrations/0001_initial.py]: True removed previously

Comment (otaiwo70) [reply backend/game/migrations/0001_initial.py]: Migration was not done but redid the migration to reference updated model my mistake

**Pull request merged**